### ISSUE TITLE
Transition to proper API

### ISSUE THREAD

**Issue by mattgd** [mattgd/UnitConverter#38]
Right now UnitConverter is more of a command line script, but it may be more useful if it could be used like a proper API.

This shouldn't be too difficult with some refactoring of the `convert_units(value, from_unit, to_unit, **args)` function, but we should also make use of exceptions where appropriate.

Please feel free to take a stab at this and/or discuss possible designs and their pros/cons below.

**Comment by sgaist** [comment]
Hi,

Do you mean turn this repository into an installable python package that offers both a command line tool and the API to use ?

**Comment by mattgd** [comment]
Yep, that's what I was thinking. I think it would generally be more useful as an API, but I would also like to keep the command line functionality around if possible.

**Comment by sgaist** [comment]
AFAIK, yes, no problem with providing both.
